Web/starter_project_iconics/data/about/about-card.json

```diff
@@ -0,0 +1,38 @@
+[
+{
+    "logo": "/img/about/a2sv-program-icons/icon-tag.png",
```

Comment:
since the path to the image file is the same except for the filename just store the image file name and use the name like "/img/about/a2sv-program-icons/{file name goes here}" in the components